tsconfig.json

```diff
@@ -0,0 +1,35 @@
+{
+  "compilerOptions": {
+    "outDir": "dist",
+    "module": "esnext",
+    "lib": [
+      "dom",
+      "esnext"
+    ],
+    "moduleResolution": "node",
+    "jsx": "react",
+    "sourceMap": true,
+    "declaration": true,
+    "esModuleInterop": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noImplicitAny": true,
+    "strictNullChecks": true,
+    "suppressImplicitAnyIndexErrors": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "allowSyntheticDefaultImports": true,
+    "target": "es5",
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "downlevelIteration": true,
+    "noEmit": true
+  },
+  "exclude": [
+    "node_modules",
+  ]
```

Comment:
I like to do also `"include": ["**/*.ts", "**/*.tsx"]`.

Reply:
does that help with resolving or speed it up or something?  Don't mind adding, but am curious why

Reply:
I think it helps with resolving the files faster. You can also specifically do `["tutor/**/*.ts", "shared/**/*.ts", "exercises/**/*.ts"]` so it specifically look for only ts files in those folders.